org.civicrm.reqangular/src/common/templates/date-picker.html

```diff
@@ -0,0 +1,42 @@
+<div class="input-group chr_uib-date-picker">
+  <input
+    type="text"
+    class="form-control"
+    uib-datepicker-popup
+    ng-readonly="datePicker.ngReadonly"
+    ng-model="datePicker.ngModel"
+    ng-disabled="datePicker.ngDisabled"
+    ng-click="datePicker.isOpen = true"
+    ng-change="datePicker.ngModelChange()"
+    show-button-bar="datePicker.showButtonBar"
+    datepicker-options="datePicker.datepickerOptions"
+    is-open="datePicker.isOpen"/>
+  <span
+    class="input-group-addon pointer"
+    ng-disabled="datePicker.ngDisabled"
+    ng-click="datePicker.customNgClick ? datePicker.customNgClick : datePicker.isOpen = true">
```

Comment:
Turns out customNgClick is not necessary at all. Previously it was there in the markup for the To Date button, because we wanted to check if the From Date is selected. But ng-disabled takes care of it. Because when the button is disabled, you cant click anyways, so no need of checking the same from ng-click.